src/formatter/android/AboutLibrariesJson.ts

```diff
@@ -0,0 +1,94 @@
+import he from 'he'
+import crypto from 'crypto'
+
+import Formatter from '../Formatter'
+import License from '../../models/License'
+
+export default class AboutLibrariesJson implements Formatter {
+  constructor(private opt: AboutLibrariesJsonOption, private writer: Writer) {}
+
+  output(licenses: License[]): void {
+    const configPath = this.opt.outputPath || 'android/config'
+    const licensesPath = `${configPath}/licenses`
+    const librariesPath = `${configPath}/libraries`
+
+    const licenseIdentifiers: string[] = []
+    licenses.forEach(license => {
+      const authorName = license.authorName
+      const name = license.libraryName
+      const version = license.version
+      const description = he.encode(license.description)
+      const licenseName = license.license ? license.license : ''
+      const libraryUniqueId = this.getLibraryUniqueId(license)
+      const licenseUniqueId = this.getLicenseUniqueId(license)
+
+      const libraryJson = {
+        uniqueId: libraryUniqueId,
+        developers: [
+          {
+            name: authorName,
+            organisationUrl: ''
+          }
+        ],
+        artifactVersion: version,
+        description,
+        name,
+        tag: '',
+        licenses: [licenseUniqueId]
+      }
+
+      const libraryJsonPath = `${librariesPath}/${libraryUniqueId}.json`
+      this.writer
+        .write(libraryJsonPath, JSON.stringify(libraryJson))
+        .then(() =>
+          console.log(`output about-libraries format to '${libraryJsonPath}'.`)
```

Comment:
Each file names are logged in console.
It becomes many if there are a lot of licenses.
I will delete this if it is not needed.